src/app/index.js

```diff
@@ -76,8 +76,9 @@ angular.module('janusHangouts', ['ngAnimate', 'ngCookies', 'ngTouch',
     if (request.status === 200) {
       var config = JSON.parse(request.responseText);
       angular.forEach(config, function(value, key) {
-         jhConfig[key] = value;
+         jhConfig[key] = value.replace("%{hostname}",window.location.host);
       });
+      console.log(jhConfig);
```

Comment:
Is this one a left-over from a debugging session? If not, just please add some description so you know what this object is. Something like this:

```javascript
console.log("Processed configuration: ", jhConfig);
```

Sure you find a better name.